src/component/route/Profile/AccountTx.jsx

```diff
@@ -104,21 +133,40 @@ const NoTx = () => (
 
 
 export default class AccountTx extends React.Component {
+  state = {
+    currentTab: TxTypes.deposit,
+  }
+
+  handleChangeTab = (event, newTab) => {
```

Comment:
Mark anything that not considered with `_`

Eg: in this case, `event` is not used, but still a positional param, we mark it with `_`
```js
handleChangeTab = (_, newTab) => ...
```